strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/users/UserControllerTestIT.java

```diff
@@ -666,99 +649,6 @@ public void testDeletingRootAdminShouldBeForbidden()
                .body(containsString(USER_DELETE_FORBIDDEN));
     }
 
-    @Test
-    @WithUserDetails("admin")
-    public void testUpdateAccessModel()
```

Comment:
we should test updating access model within role

Reply:
there is `AuthorizationConfigControllerTestIT`, anyway I think there will be new API required for UI and this should be a follow up task, so there will be new tests